README.md

```diff
@@ -77,14 +77,34 @@ The packages that Instana offers depend on:
 
   The output above shows that the module version 1.21.3 is required for NGINX Plus R25 P1.
 
+- The OpenSSL version, as shown by the openssl version command:
+
+  ```sh
+  # openssl version
+  OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)
+  ...
+  ```
+
+  The output above shows that the module version 3.0.2 is required for libinstana_sensor_ssl3x.so.
+
+  ```sh
+  # openssl version
+  OpenSSL 1.1.1f  31 Mar 2020
+  ...
+  ```
+
+  The output above shows that the module version 1.1.1f is required for libinstana_sensor_ssl1.1x.so.
+
 - The Libc variant used in your distribution (`glibc` or `musl`); you likely use `glibc`, unless you are using Alpine as base-image for your containers, in which case, it's `musl`.
 - (In some cases) the particular distribution (when the build used in some official packages is different enough to require bespoke adjustments on Instana side)
+- The OpenSSL variant of libinstana_sensor.so (either libinstana_sensor_ssl1.1x.so or libinstana_sensor_ssl3x.so) is to support serverless monitoring with OpenSSL 1.1.x and OpenSSL 3.x.x. Choose correct OpenSSL variant for serverless monitoring; otherwise use libinstana_sensor.so.
```

Comment:
```suggestion
- The OpenSSL variant of `libinstana_sensor.so` (either `libinstana_sensor_ssl1.1x.so` or `libinstana_sensor_ssl3x.so`) is to support serverless tracing with OpenSSL 1.1.x or OpenSSL 3.x.x. Choose the correct OpenSSL variant for serverless tracing. Otherwise, use `libinstana_sensor.so`.
```

Reply:
Updated.